test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -435,3 +456,82 @@ test.requestHooks(mockEnrollAuthenticatorWithCustomApp)('should load select auth
   await t.expect(selectFactorPage.getFactorSelectButtonDataSeByIndex(4, true)).eql('webauthn');
   await t.expect(await selectFactorPage.factorUsageTextExistsByIndex(4)).eql(false);
 });
+
+// re-enable with gen 3 changes
+test.meta('gen3', false).requestHooks(mockEnrollRequiredNowAndRequiredSoonAuthenticators)('should load select required now and required soon authenticator lists', async t => {
```

Comment:
@jakebacher-okta you'll likely want to enable gen3 here and update the selectors in the page object to run these test for v3 parity in your siw v3 changes